test/system/045-start.bats

```diff
@@ -59,4 +59,12 @@ load helpers
     is "$output" "Error: fakepolicy invalid restart policy"
 }
 
+@test "podman start --all --filter" {
+    run_podman run -d $IMAGE /bin/true
+    cid="$output"
+    run_podman run -d $IMAGE /bin/false
+    run_podman start --all --filter exited=0
+    is "$output" "$cid"
```

Comment:
Should we `run_podman rm $cid` here, @edsantiago ?

Reply:
It's OK not to: by now there are so many stray containers in tests everywhere, including this file itself, that I'm OK with not cleaning up containers. (Stray images, though, I get cranky about).

What I will insist on is `run_podman wait $cid AND-CID-OF-THE-BIN-FALSE-CONTAINER`. @boaz0 please log that CID (or give the container a name), and add that `wait`.

Reply:
No problem @edsantiago . Thanks for your input.